test/test_helper.rb

```diff
@@ -3,6 +3,9 @@
 # Configure Rails Environment
 ENV["RAILS_ENV"] = "test"
 
+# Don't wrap output in tests
+ENV["THOR_COLUMNS"] = "200"
```

Comment:
This is more resilient than the old `Thor::Base.shell.any_instance.stubs(:terminal_width).returns(200)`, since `THOR_COLUMNS` has been around for more than 10 years, and it still works with Thor 1.3.0.